### ISSUE TITLE
Create a grid source space

### ISSUE THREAD

**Issue by delshadv** [OHBA-analysis/osl-core#55]
Hi @jhadida @GilesColclough @woolrich and other OSL team,

Hope you’re doing well!!
I need to do source-space analyses using a volumetric regular grid that is
generated from the MNI template. (Exactly like analysis 3 of @woolrich BioFind paper.)
I was wondering If you could let me know that this source reconstruction using a grid are available in OSL? or does OSL have any solution for this regard?

Thanks
Delshad

**Comment by delshadv** [comment]
Thanks @jhadida - I've already used your solution as we started to use OSL for source and ROI analysis but the results are somewhat different. We think it might related to this point:
In the mentioned paper a volumetric and regular grid with **1‐cm spacing** was generated for the MNI template. (Only sources inside the brain surface were taken into account, resulting in a source model with 2459 sources, each consisting in three perpendicular dipoles. Anatomical labels were assigned to each source according to reduced AAL version of the Harvard‐Oxford atlas.)
